runtime-android/authentication-challenge/authentication-challenge/app/src/main/java/com/thosedevs/authenticationchallenger/MainActivity.kt

```diff
@@ -0,0 +1,116 @@
+package com.thosedevs.authenticationchallenger
+
+import android.app.Activity
+import android.app.AlertDialog
+import android.app.Dialog
+import android.app.DialogFragment
+import android.content.Context
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
+import com.esri.arcgisruntime.data.ArcGISFeatureTable
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.security.*
+import kotlinx.android.synthetic.main.activity_main.*
+import android.view.LayoutInflater
+import android.widget.EditText
+import android.widget.Toast
+import kotlinx.android.synthetic.main.login_dialog_layout.*
+import java.util.concurrent.CountDownLatch
+
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        //Uncomment to see default behavior
+        //setUpDefaultChallengeHandler()
+
+        AuthenticationManager.setAuthenticationChallengeHandler(CustomAuthenticationChallengeHandler(this))
```

Comment:
@nohe427 , could we add another comment here to comment out this line when uncommenting above to view default behavior?

Reply:
Updated